mcs/class/System/Mono.Net.Security/NoReflectionHelper.platformnotsupported.cs

```diff
@@ -0,0 +1,99 @@
+//
```

Comment:
We should be able to get rid of this (and the other `NoReflectionHelper.cs`) once the refactoring is done.

This file is currently needed because some of the types it's using are not defined at the time `Mono.Security.dll` is built.  Once we have "platformnotsupported.cs" versions for them, we can get rid of this and the ugly `(object)` casts.